tox.ini

```diff
@@ -20,10 +22,12 @@ commands =
 [testenv:gui35]
 basepython=python3.5
 passenv = TOXENV DISPLAY HOME
+whitelist_externals = bash py.test
 commands =  
            # hack since enable is not packaged correctly
            pip install -r{toxinidir}/requirements_gui.txt
            pip install pyqt5==5.10.0
+           bash {toxinidir}/scripts/font_manager.bash
```

Comment:
what kind of an issue is this hack for?

Reply:
font manager spews a bunch of exceptions on start-up while it's building it's cache, it doesn't look like it's really an error, but it creates a lot of noisy logging, this squelches it

Reply:
maybe add a comment about it?

Reply:
Good idea, added comments